redash/saml_auth.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2015 Okta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Is this legal note really needed here, or was it added by your IDE automatically or something similar?

Otherwise I need to check implications. It's probably non issue, as it's the same as including any other open source library, but IANAL...